### PR TITLE
CSEN-11 Scala style checker was added into project

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# cs-orm
+# cs-entities
 Simple CloudStack ORM
```
Comment (placccebo) [inline README.md]: replace with "Simple CloudStack Entities Framework"

**Review by placccebo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MedvedevBW) [reply README.md]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**